extensions/amp-story/1.0/amp-story-reaction.js

```diff
@@ -178,16 +188,24 @@ export class AmpStoryReaction extends AMP.BaseElement {
   }
 
   /** @override */
-  buildCallback() {
+  buildCallback(concreteCSS = '') {
     this.options_ = this.parseOptions_();
     this.rootEl_ = this.buildComponent();
+    this.rootEl_.classList.add('i-amphtml-story-reaction');
```

Comment:
This will cause issues on browsers without shadow DOM, or environments where shadow DOM is disabled. Let's change the class name so it's different for `rootEl_` and `element`

Reply:
The main use for the this.element having a class was to set the pointer events to none. SInce that's gone, we can remove it from here for now